Google/Makefile

```diff
@@ -43,10 +43,18 @@ generate_credentials:
 	./fetch_config.rb -s azure-key-vault-secret:${KEY_VAULT}/${ASSET_SECRETS}  -f shell-env-var -d command -- jinja2 creds.json.tmpl > creds.json
 	clasp login --creds creds.json
 
+storage_parameters:
+	./fetch_config.rb -s azure-key-vault-secret:${KEY_VAULT}/${ASSET_SECRETS}  -f shell-env-var -d command -- jinja2 ../azure/storage_parameters.json.tmpl > ../azure/storage_parameters.json
```

Comment:
I've tested a bash based templating solution to avoid the dependency on jinja2, pip and python. Please try:
```
./fetch_config.rb -s azure-key-vault-secret:${KEY_VAULT}/${ASSET_SECRETS}  -f shell-env-var -d command -- envsubst \'\$${STORAGE_NAME}\' < ../azure/storage_parameters.json.tmpl
```

Reply:
Now using `erb`